pkg/tracer/tcptracer-ebpf.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
> But then you will need to recommit it every time the C-code is modified, right?

Right

> Will patches be reasonable? (i.e. if only the _tcptracerEbpfO changes, then that's OK)

I tried to change the ebpf source and check the diff.
- When the change in the source does not change the result (i.e. whitespace / indentation), then there is no diff at all because we compile with `go-bindata -modtime 1` to have constant modification timestamp.
- With other changes in the source, I get a change in `_tcptracerEbpfO` and a change in the "size" metadata like this:

```patch
-       info := bindataFileInfo{name: "tcptracer-ebpf.o", size: 18848, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+       info := bindataFileInfo{name: "tcptracer-ebpf.o", size: 18856, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
```

If a developer forgets to commit the change in `pkg/tracer/tcptracer-ebpf.go`, then CircleCI will error out.